src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -406,7 +404,7 @@ public partial class ProjectResourceOptions
 
     public static partial class PublisherDistributedApplicationBuilderExtensions
     {
-        [System.Diagnostics.CodeAnalysis.Experimental("ASPIREPUBLISHERS001")]
+        [System.Diagnostics.CodeAnalysis.Experimental("ASPIREPUBLISHERS001", UrlFormat = "https://aka.ms/aspire/diagnostics/{0}")]
         public static IDistributedApplicationBuilder AddPublisher<TPublisher, TPublisherOptions>(this IDistributedApplicationBuilder builder, string name, System.Action<TPublisherOptions>? configureOptions = null)
```

Comment:
Does this need to be public anymore? We only expect 2 publishers - manifest and default, right?

Reply:
It doesn't.

Reply:
https://github.com/dotnet/aspire/pull/9285